README.md

```diff
@@ -46,27 +46,22 @@ There are several steps to importing new content into the prototype:
 0. Tag all the content to taxons using [content
```

Comment:
Could we also add 2 notes on installing dependencies? Something like:

Install ruby dependencies with:

```
bundle install
```

Install python dependencies:

```
pip install -r requirements.txt
```

Just for completeness.